src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaGitContainer/Dockerfile

```
@@ -3,15 +3,16 @@
 # and prepares for execution of Java compile jobs on slaves.
 #
 
-FROM ubuntu:xenial
+FROM ubuntu:jammy
```

any thoughts on using the version number instead which is easier than remembering each codename or googling it?
```suggestion
FROM ubuntu:22.04
```

---

I have no objection